docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -775,9 +775,78 @@ See below for a visual representation:
 
 **Birth Weight** 
 
-.. todo:: 
+Our model will apply the effect size of maternal consumption of iron fortified 
+foods on infant birth weight under the following assumptions:
+
+1. The effect of consuming iron fortified foods takes place six months after 
+the onset of consumption. This assumption is partially supported by Andersson 
+et al. 2010, which found that the peak response occurs at approximately six 
+months after onset; however, we simplified this finding for our model and 
+assumed that there was only a response six months after onset of consumpiton 
+of iron fortified foods.
+
+2. The effect of maternal iron fortified food consumption must occur at the 
+start of the third trimester of pregnancy in order to impact infant birth 
+weight, which we assumed was equal to 3 months prior to birth of the infant. 
+We chose to make this assumption because the effect of maternal iron was 
+measured at the third trimester in the Haider et al. 2013 paper.
+
+Therefore, for our simulantion, an infant's mother must have gained coverage 
+to iron fortified foods at least **nine months prior to the birth of the 
+infant** in order for the iron fortification coverage to affect the infant's 
+birth weight.
+
+.. todo::
+
+  Confirm that this is the best approach by evaluating the individual studies 
+  cited in the Haider et al. 2013 paper and update as necessary (as discussed 
+  in `PR #181 <https://github.com/ihmeuw/vivarium_research/pull/181>`_).
+
+.. note:: 
 
-	Write this section
+  The following was adopted from Nathaniel's description for folic acid.
+
+
+1.  **Define variables:** Each simulant needs an attribute
+    `mother_ate_iron_fortified_food`, which will be `True` if the simulant's
+    mother ate iron fortified food starting at least nine months (0.75 years) 
+    before the simulant was born, `False` if not, and `Unknown` if we don't 
+    know.
+
+2.  **Initialize the simulation:** At the start of the simulation, set
+    `mother_ate_iron_fortified_food = Unknown` for all simulants in age groups 
+    4 and 5; this attribute will not be used for these age groups because the 
```

Comment:
I think it would be helpful to state the actual ages of the simulants here and/or include the descriptive names of the age groups (e.g. post-neonatal and ages 1-4?). I don't remember off-hand what age groups 4 and 5 are.

Reply:
Updated